app/models/rental.py

```diff
@@ -0,0 +1,31 @@
+from flask import current_app
+from app import db
+from datetime import datetime, timedelta
+
+
+class Rental(db.Model):
+    __tablename__ = 'rentals'
+
+    rental_id = db.Column(db.Integer, primary_key=True)
+
+    customer_id = db.Column(
+        db.Integer,
+        db.ForeignKey('customers.customer_id'))
+
+    video_id = db.Column(
+        db.Integer,
+        db.ForeignKey('videos.video_id'))
+
+    due_date = db.Column(
+        db.DateTime,
+        default=(datetime.today() + timedelta(days=7)).strftime("%Y-%m-%d")
+    )
```

Comment:
This is just fine! But there's a lot of extra space here, and it makes it harder to read.

```suggestion
    __tablename__ = 'rentals'
    rental_id = db.Column(db.Integer, primary_key=True)
    customer_id = db.Column(db.Integer, db.ForeignKey('customers.customer_id'))
    video_id = db.Column(db.Integer, db.ForeignKey('videos.video_id'))
    due_date = db.Column(
        db.DateTime,
        default=(datetime.today() + timedelta(days=7)).strftime("%Y-%m-%d")
    )
```